### PR TITLE
Feature/installation step by step

### PR DESCRIPTION
**Pull request by sensorario**
| question | answer |
|---|---|
| backward compatibility? | yes|
| new feature? |no |
| bug fix? | yes|
| new deprecations? | no |

This PS aims to list all installation steps in readme file.

- [x] fix typos
- [x] squash everything
- [ ] merge
- [ ] release new tag

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+# Plain symfony project for query-bundle
+
+The purpose of this project is to see how studiomado/query-bundle works and can be installed in a plain symfony project.
+
+ - database condifuration
+ - install query-bundle
+ - create at least one entity
+
+# database condiguration
```
Comment (AlessandroMinoccheri) [inline README.md]: typo "configuration"

File: README.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+Rember to update parameter.yml file, parameter.yml.dist and config.yml file. In config.yml file also remember that the drive MUST be changed in pdo_sqlite to enable doctrine to work with this database.
```
Comment (AlessandroMinoccheri) [inline README.md]: "Remember" not "Rember"

File: README.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+    composer require white-october/pagerfanta-bundle
+
+and then, … add vendors in your app/AppKernel 
+
+    new JMS\SerializerBundle\JMSSerializerBundle(),
+    new Bazinga\Bundle\HateoasBundle\BazingaHateoasBundle(),
+
+# Complete configuration and use the bundle
+
+Once everything is done, you can, you can add new endpoints using the query-bundle to query the database.
```
Comment (AlessandroMinoccheri) [inline README.md]: "you can" repeated

**Comment by sensorario** [comment]
@AlessandroMinoccheri LGTY?

**Comment by AlessandroMinoccheri** [comment]
LGTM

**Pull request merged**